evergreen/convert-client-cert-to-pfx.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with an error if any of the commands fail
+
+# Environment variables used as input:
+#   CLIENT_PEM                      Path to mongo -orchestration's client.pem: must be set.
+#   MONGO_X509_CLIENT_P12           filename for client certificate in p12 format
+#
+# Enviornment variables produced as output:
+#   MONGODB_X509_CLIENT_P12_PATH    Absolute path to client certificate in p12 format
+#   MONGO_X509_CLIENT_P12_PASSWORD  password for client certificate
+
+
+CLIENT_PEM=${CLIENT_PEM:-nil}
+MONGO_X509_CLIENT_P12=${MONGO_X509_CLIENT_P12:-client.p12}
+MONGO_X509_CLIENT_P12_PASSWORD=${MONGO_X509_CLIENT_P12_PASSWORD:-Picard-Alpha-Alpha-3-0-5}
+
+if [[ "$CLIENT_PEM" == "nil" ]]; then
+  exit 1
+fi
+
+openssl pkcs12 -export -in "${CLIENT_PEM}" \
+  -out "${MONGO_X509_CLIENT_P12}" \
+  -name "Drivers Client Certificate" \
+  -password "pass:${MONGO_X509_CLIENT_P12_PASSWORD}"
```

Comment:
`pass:` is NOT part of the password, but is a required part of the parameter

Reply:
Brief research shows that it's technically possible to create a `.p12` file without a password, but I don't think `openssl` can do it. From my brief bit of research, I believe `p12` files with passwords will be the more common use case.